raiden/network/transport/matrix/transport.py

```diff
@@ -440,7 +440,9 @@ def start(  # type: ignore
         self._initialize_first_sync()
         self._initialize_room_inventory()
         self._initialize_broadcast_rooms()
-        self._initialize_whitelist(whitelist)
+
+        self._schedule_new_greenlet(self._health_check_worker)
```

Comment:
I think we don't even have to move the worker up there. Is that correct @hackaugusto ?

Reply:
Yeah, it shouldn't matter that much.